README.md

```diff
@@ -1264,6 +1319,35 @@ You can set a timeout for retrying the operations sent to a member. This can be
 
+When invocation is being retried, the client may wait some time before it retries again. This retry wait time is configurable with this configuration property:
+
+```
+config.setProperty(“hazelcast.client.invocation.retry.pause.millis”, “500");
+```
+The default retry wait time is 1 second.
+
+## 7.3.3 Client Backpressure
+
+Hazelcast uses operations to make remote calls. For example, a `map.get` is an operation and a `map.put` is one operation for the primary
+and one operation for each of the backups, i.e. `map.put` is executed for the primary and also for each backup. In most cases, there will be a natural balance between the number of threads performing operations
+and the number of operations being executed. However, there are two situations where this balance and operations
+can pile up and eventually lead to Out of Memory Exception (OOME):
+
+- Asynchronous calls: With async calls, the system may be flooded with the requests.
+- Asynchronous backups: The asynchronous backups may be piling up.
+
+To prevent the system from crashing, Hazelcast provides back pressure. Back pressure works by:
+
+- limiting the number of concurrent operation invocations,
+- periodically making an async backup sync.
+
+Sometimes, e.g., when your servers are overloaded, you may want to slow down the client operations to the cluster. Then the client can be configured to wait until number of outstanding invocations whose reponses are not received to become less than a certain number. This is called "Client Backpressure". By default, the backpressure is disabled. There are a few properties which control the backpressure. These client configuration properties are:
+
+- `hazelcast.client.max.concurrent.invocations`: The maximum number of concurrent invocations allowed. To prevent the system from overloading, you can apply a constraint on the number of concurrent invocations. If the maximum number of concurrent invocations has been exceeded and a new invocation comes in, then Hazelcast will throw HazelcastOverloadException. By default this property is configured as INT32_MAX.
+- `hazelcast.client.invocation.backoff.timeout.millis`: Controls the maximum timeout in millis to wait for an invocation space to be available. If an invocation can't be made because there are too many pending invocations, then an exponential backoff is done to give the system time to deal with the backlog of invocations. This property controls how long an invocation is allowed to wait before getting `HazelcastOverloadException`. When set to -1 then `HazelcastOverloadException` is thrown immediately without any waiting. This is the default value.
+
+For details of backpressure, see the [Backpressure section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#back-pressure) in the Hazelcast IMDG Reference Manual.
```

Comment:
For more information, see the [Back Pressure section](https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#back-pressure) in the Hazelcast IMDG Reference Manual.